src/main/java/io/cdap/plugin/gcp/bigquery/sink/BigQuerySinkUtils.java

```diff
@@ -279,6 +279,34 @@ public static String configureBucket(Configuration baseConfiguration,
     baseConfiguration.setBoolean("fs.gs.metadata.cache.enable", false);
     return bucket;
   }
+  /**
+   * Sets string fields to JSON type if they are present in the provided list of fields.
+   * @param fields list of BigQuery table fields.
+   * @param jsonStringFields Comma separated list of fields that should be set to JSON type.
+   *
+   */
+  public static void setJsonStringFields(List<BigQueryTableFieldSchema> fields,
+                                                                   String jsonStringFields) {
+    Set<String> jsonFields = new HashSet<>(Arrays.asList(jsonStringFields.split(",")));
+    setJsonStringFields(fields, jsonFields, new ArrayList<>());
+  }
+
+  private static void setJsonStringFields(List<BigQueryTableFieldSchema> fields, Set<String> jsonFields,
+                                             List<String> path) {
+    for (BigQueryTableFieldSchema field : fields) {
+      String fieldName = field.getName();
+      String fieldType = field.getType();
+      String separator = path.isEmpty() ? "" : ".";
+      String fieldPath = String.join(".", path) + separator + fieldName;
+      if (jsonFields.contains(fieldPath) && fieldType.equals(LegacySQLTypeName.STRING.name())) {
+        field.setType(LegacySQLTypeName.valueOf("JSON").name());
```

Comment:
add a  `private static final String JSON = "JSON"`; 
and then use it use.

Reply:
Added new 
```java 
private static final String JSON = "JSON"; 
```